app/views/components/back_link.scala.html

```diff
@@ -14,8 +14,18 @@
  * limitations under the License.
  *@
 
-@(href: String)(implicit messages: Messages)
+@import models.Mode
+@import models.Mode._
+ 
+@(href: Call, mode: Mode = Mode.Normal)(implicit messages: Messages)
```

Comment:
👍

Reply:
Do we need this default parameter? In practice probably always we're passing this value to the view

Reply:
Mode is only required on the journey pages, other pages e.g. Submissions/Drafts all have back buttons too and dont care about the mode